config/playground/helm/prod/helm-opensearch.yaml

```diff
@@ -30,6 +30,7 @@ opensearchHome: /usr/share/opensearch
 config:
   opensearch.yml: |
     cluster.name: opensearch-cluster
+    plugins.query.datasources.encryption.masterkey: ${QUERY_DATASOURCES_MASTER_KEY}
```

Comment:
I saw that you add parameter is not required, is it meaning that this value can be null in the opensearch.yaml? is it blocking opensearch startup or just ignore.

Reply:
If someone doesn't want to use metrics, then this is not required. Since the template is used by other folks as well, I made it as optional.